cypress/README.md

```diff
@@ -15,3 +15,45 @@ Or run `yarn cy:ci` to run all tests.
 - Since our application is relatively simple, there's no need for Page Objects yet
 - [Don't write small tests with single assertions](https://docs.cypress.io/guides/references/best-practices#Creating-tiny-tests-with-a-single-assertion)
 - [Mock HTTP requests](https://docs.cypress.io/guides/guides/network-requests#Stub-Responses) when appropriate
+
+# Visual testing
+## tl;dr
+To update the snapshots, run
+```
+yarn cy:ss
+```
+(`ss` stands for **screenshot**)
+
+It requires docker installed and available under the `docker` binary in `PATH`.
+(Not tested with `podman` or other alternatives)
+
+To just run without updating the snapshots, run
+```
+CYPRESS_updateSnapshots=false yarn cy:ss
+```
+
+## Why
+Part of our core functionality revolves around rendering a flamegraph **canvas**,
+which is not straightforward to test.
+
+We've decided to test it using [visual testing](https://docs.cypress.io/guides/tooling/visual-testing#Functional-vs-visual-testing).
```

Comment:
So am I correct in understanding you set a threshold somewhere of how different the pictures can be before failing the test?

Reply:
That is correct. However, I didn't set anything, so it's using the plugin's default, [which is 0](https://github.com/jaredpalmer/cypress-image-snapshot/blob/0510252bc5f192c410bcca45b511f07737bce4a2/src/plugin.js#L61).